bcdaworker/queueing/enqueue.go

```diff
@@ -24,16 +25,16 @@ type Enqueuer interface {
 
 // Creates a river client for the Job queue. If the client does not call .Start(), then it is insert only
 // We still need the workers and the types of workers to insert them
-func NewEnqueuer() Enqueuer {
+func NewEnqueuer(db *sql.DB, pool *pgxv5Pool.Pool) Enqueuer {
```

Comment:
@carlpartridge here the pool is passed down to the the enqueuer via its constructor and subsequently to the river client that it stores in its state.